src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -0,0 +1,173 @@
+@import '~amo/css/styles';
+
+.HeroRecommendation {
+  color: $white-100;
+  display: flex;
+  padding: 24px;
+  position: relative;
+
+  @include respond-to(extraExtraLarge) {
+    padding: 36px;
+
+    @include padding-start(96px);
+  }
+}
+
+.HeroRecommendation-info {
+  @include margin-start(24px);
+
+  @include respond-to(extraLarge) {
+    @include margin-start(96px);
+  }
+
+  @include respond-to(extraExtraLarge) {
+    @include margin-start(136px);
+  }
+}
+
+$overlayShapeZIndex: 1;
+
+.HeroRecommendation-overlayShape {
+  bottom: 0;
+  height: 70%;
+  left: 0;
+  max-height: 307px;
+  position: absolute;
+  z-index: $overlayShapeZIndex;
+
+  [dir='rtl'] & {
+    left: auto;
+    right: 0;
+    transform: scaleX(-1);
+  }
+}
+
+.HeroRecommendation-image {
+  height: auto;
+  width: 100%;
+
+  @include respond-to(extraExtraLarge) {
+    width: 480px;
+  }
+}
+
+.HeroRecommendation-recommended {
+  font-size: $font-size-s;
+  letter-spacing: 0.1em;
+  line-height: 1.143;
+  margin: 0;
+  opacity: 0.5;
+  text-transform: uppercase;
+
+  @include respond-to(medium) {
+    font-size: $font-size-default;
+  }
+
+  @include respond-to(extraExtraLarge) {
+    margin-top: 16px;
+  }
+}
+
+.HeroRecommendation-heading {
+  color: $white-100;
+  font-size: $font-size-l;
+  line-height: 1.188;
```

Comment:
That's very precise, heh.

Reply:
I just calculated them from the Sketch file 🤷‍♂